### PR TITLE
Extended RB overview 

### PR DESCRIPTION
**Pull request by ShellyGarion**
### Summary
- Extended RB overview to contain more explanation on how RB works and simultaneous RB
- Minor changes in Clifford_Group.ipnyb (added logo)

@dcmckayibm - please review

### CODE REVIEW

File: community/ignis/RB_overview.ipynb [hunk 1]
```diff
@@ -287,35 +291,178 @@
...
+    "*Note:* In the graph above $p$ is called \"alpha\"."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### The intuition behind RB\n",
+    "\n",
+    "The depolarizing channel has parameter $p$, and works like this: with probability $p$, the state remains the same as before; with probability $1-p$, the state becomes the totally mixed state.\n",
```
Comment (dcmckayibm) [inline community/ignis/RB_overview.ipynb]: If you look at our papers we use $\alpha$ for $p$ here. Also you have $1-p^m$ should be $(1-p)^m$. The last equation didn't render.

**Comment by dcmckayibm** [comment]
I think the intuition paragraph would be better if you just wrote out the quantum channel

$\rho_f = \alpha \rho_i + (1-\alpha)/2^n * \mathbf{I} $

$\rho_f^m = \alpha^m \rho_i + (1-\alpha)^m/2^n * \mathbf{I} $

Then it's clear that the ground state survival probability just follows.

**Comment by ShellyGarion** [comment]
@dcmckayibm - I updated the overview following your comments above.

**Comment by dcmckayibm** [comment]
Introduction is spelled wrong

**Comment by dcmckayibm** [comment]
This sentence is a little confusing "Suppose that we have a sequence of $m$ gates, not necessarily Clifford gates. Each gate is affected by the depolarizing channel with parameter $\alpha$ (same $\alpha$ for all the gates)." 

-> I would say that suppose we have gates where the error channel of the gate is a depolarizing channel. 
-> I was also wrong it's not (1-alpha)^m it's (1-alpha^m)...then the last equation is exact

**Comment by ShellyGarion** [comment]
@dcmckayibm - I updated the overview again following your comments above.

**Pull request merged**